sihl/src/index.mld

```diff
@@ -12,6 +12,12 @@ Sihl is a batteries-included web framework built on top of {{: https://github.co
 
 {1 Getting Started}
 
+{2 Prerequisites}
+
+You need to have {{:https://opam.ocaml.org/doc/Install.html} OPAM} installed and you should know OCaml somewhat.
+
+I is also recommended that you somewhat know how to use OCaml, its standard library and [Lwt.t]. This documentation explains a lot of concepts in terms of types, so you should be comfortable reading type signatures. If you are beginner, check out the section {{!learn_ocaml} OCaml for web development in Sihl} to learn enough to be dangerous.
```

Comment:
- It* is also
- you are a* beginner

The first sentence is a bit redundant considering the text before has the same text.

Reply:
Fixed.